yarn.lock

```diff
@@ -4893,7 +4893,7 @@ loglevel-colored-level-prefix@^1.0.0:
 
 loglevel@^1.4.1:
   version "1.5.1"
```

Comment:
If anyone knows how I can get rid of the entry in the yarn file, I'd be eternally grateful.

Reply:
TIL `loglevel` is a dependency of `loglevel-colored-level-prefix`, which is a depeddency of `prettier-eslint-cli`